external/fv3fit/fv3fit/sklearn/_wrapper.py

```diff
@@ -32,6 +33,58 @@ def _tuple_to_multiindex(d: tuple) -> pd.MultiIndex:
     return pd.MultiIndex.from_tuples(list_, names=names)
 
 
+@_shared.io.register("sklearn")
+@register_estimator("sklearn", RandomForestHyperparameters)
+@register_estimator("rf", RandomForestHyperparameters)
+@register_estimator("random_forest", RandomForestHyperparameters)
+@register_estimator("sklearn_random_forest", RandomForestHyperparameters)
+class RandomForest(Estimator):
+    def __init__(
+        self,
+        sample_dim_name: str,
+        input_variables: Iterable[str],
+        output_variables: Iterable[str],
+        hyperparameters: RandomForestHyperparameters,
+    ):
+        batch_regressor = RegressorEnsemble(
+            sklearn.ensemble.RandomForestRegressor(
+                n_jobs=hyperparameters.n_jobs,
```

Comment:
I think in the current use case (using `RegressorEnsemble` as a hacky forest and 1 tree per sklearn RF) neither the higher-up `SklearnWrapper` n_jobs nor the sklearn model hyperparameter n_jobs are doing any parallelization of subsetted data after splits. Since there's no difference between picking one of those to set right now I think it would be easier for the future PR that removes the `RegressorEnsemble` to have `n_jobs` set as the `RandomForestRegressor` hyperparameter in this PR, so that this code doesn't have to get adjusted later. At least I think that is how the parallelization is currently working (or not working).

Reply:
Isn't the current parallel backend context in `predict` parallelizing calls over the RFs in the `RegressorEnsemble` if `n_jobs = -1`?

Reply:
OK as discussed it's not actually doing any parallelism currently so we can delete that context and follow @AnnaKwa's suggestion